source/octf/utils/table/Cell.h

```diff
@@ -117,6 +117,14 @@ class Cell {
                 : IData()
                 , m_value(other.m_value) {}
 
+        Data<T> &operator=(const Data<T> &other) {
+            if (&other == this) {
+                return *this;
+            }
+
+            return other->duplicate();
```

Comment:
Here we need to modify m_data. And return references to this object.

Reply:
Please update this.

Reply:
Updated